packages/react-table/src/demos/examples/ComposableTableStickyToolbar.tsx

```diff
@@ -0,0 +1,268 @@
+                    <Th isStickyColumn modifier="truncate" sort={getSortParams(0)}>
+                      {columnNames.name}
+                    </Th>
+                    <Th
+                      isStickyColumn
+                      stickyMinWidth="120px"
+                      stickyLeftOffset="120px"
+                      hasRightBorder
+                      modifier="truncate"
+                      sort={getSortParams(1)}
+                    >
+                      {columnNames.state}
+                    </Th>
+                    <Th modifier="truncate">{columnNames.header3}</Th>
+                    <Th modifier="truncate">{columnNames.header4}</Th>
+                    <Th modifier="truncate">{columnNames.header5}</Th>
+                    <Th modifier="truncate">{columnNames.header6}</Th>
+                    <Th modifier="truncate">{columnNames.header7}</Th>
+                    <Th modifier="truncate">{columnNames.header8}</Th>
+                    <Th modifier="truncate">{columnNames.header9}</Th>
+                  </Tr>
+                </Thead>
+                <Tbody>
+                  {sortedFacts.map(fact => (
+                    <Tr key={fact.name}>
+                      <Th isStickyColumn modifier="truncate">
+                        {fact.name}
+                      </Th>
+                      <Th
+                        isStickyColumn
+                        stickyMinWidth="120px"
+                        stickyLeftOffset="120px"
+                        modifier="truncate"
+                        hasRightBorder
+                      >
+                        <BlueprintIcon />
+                        {` ${fact.state}`}
+                      </Th>
+                      <Td modifier="nowrap" dataLabel={columnNames.header3}>
+                        {fact.detail1}
+                      </Td>
+                      <Td modifier="nowrap" dataLabel={columnNames.header4}>
+                        {fact.detail2}
+                      </Td>
+                      <Td modifier="nowrap" dataLabel={columnNames.header5}>
+                        {fact.detail3}
+                      </Td>
+                      <Td modifier="nowrap" dataLabel={columnNames.header6}>
+                        {fact.detail4}
+                      </Td>
+                      <Td modifier="nowrap" dataLabel={columnNames.header7}>
+                        {fact.detail5}
+                      </Td>
+                      <Td modifier="nowrap" dataLabel={columnNames.header8}>
+                        {fact.detail6}
+                      </Td>
+                      <Td modifier="nowrap" dataLabel={columnNames.header9}>
+                        {fact.detail7}
+                      </Td>
+                    </Tr>
+                  ))}
+                </Tbody>
+              </TableComposable>
+            </InnerScrollContainer>
+          </OuterScrollContainer>
+        </div>
+        <br />
+        Other content
```

Comment:
are these "Other content" line needed?

Reply:
I can remove them. I had added them initially to show the page scrolling but that may also be seen by making the window shorter.